infra/ansible/roles/nginx/tasks/main.yml

```diff
@@ -54,25 +54,11 @@
     install_recommends: no
     update_cache: yes
 
-- name: Restrict TLS ssl_protocols
-  lineinfile:
-    path: /etc/nginx/nginx.conf
-    regexp: '^\s*ssl_protocols'
-    line: "\t\tssl_protocols TLSv1.2;"
-  notify:
-    - Reload Nginx
-
-- name: Add TLS parameters
-  blockinfile:
-    path: /etc/nginx/nginx.conf
-    insertafter: '^\s*ssl_prefer_server_ciphers'
-    block: |
-      ssl_ecdh_curve secp384r1;
-      ssl_ciphers 'ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-SHA384:ECDHE-RSA-AES256-SHA384:ECDHE-ECDSA-AES128-SHA256:ECDHE-RSA-AES128-SHA256';
-      ssl_session_cache shared:SSL:10m;
-      ssl_session_tickets off;
-  notify:
-    - Reload Nginx
+- name: Set Nginx configuration
+  template:
+    src: nginx.conf.j2
+    dest: /etc/nginx/nginx.conf
+  notify: Reload Nginx
```

Comment:
instead of modifying the default configuration, I set a static configuration that is just copied.